scripts/opencontrail-install/contrail_install.rb

```diff
@@ -5,20 +5,36 @@
 
 raise 'Must run with superuser privilages' unless Process.uid == 0
 
+#TODO: Add all these through proper command line options
+@controller_host = ARGV[0]
+@role = ARGV[1]
+@setup_kubernetes = true
+
+@private_net = "10.10.0.0/16"
+@portal_net = "10.0.0.0/16"
```

Comment:
In my install /etc/kubernetes/apiserver contains:
# Address range to use for services
KUBE_SERVICE_ADDRESSES="--portal_net=10.254.0.0/16"

I'm not sure whether this is the default or whether it was setup by the ansible provisioning; regardless the parameters must match (we should actually just scan the kubernetes config files).